src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3883,12 +3894,40 @@ private VisitArgumentResult VisitArgumentEvaluate(BoundExpression argument, RefK
             Debug.Assert(!this.IsConditionalState);
         }
 
-        private void CheckDisallowedNullAssignment(TypeWithState state, FlowAnalysisAnnotations annotations, Location location)
+        private void CheckDisallowedNullAssignment(TypeWithState state, FlowAnalysisAnnotations annotations, Location location, BoundExpression boundValueOpt = null)
         {
-            if (((annotations & FlowAnalysisAnnotations.DisallowNull) != 0) && state.Type.IsNullableTypeOrTypeParameter() && state.MayBeNull)
+            if (boundValueOpt is object && boundValueOpt.WasCompilerGenerated)
+            {
+                // We need to skip `return backingField;` in auto-prop getters
```

Comment:
>// We need to skip `return backingField;` in auto-prop getters [](start = 16, length = 62)

Can we simply avoid running `NullableWalker` on auto-property accessors?

Reply:
We could, but I'd prefer to contain the scope of this PR given timeline.
Also, even if we skip auto-prop accessors, we'll still need some logic to skip compiler-generated nodes because of `field = initializerValue;` that is injected to constructor body and analyzed.

---
In reply to: [367228167](https://github.com/dotnet/roslyn/pull/40789#discussion_r367228167) [](ancestors = 367228167)

Reply:
Please consider logging a bug to skip auto-property accessors.

---
In reply to: [367632535](https://github.com/dotnet/roslyn/pull/40789#discussion_r367632535) [](ancestors = 367632535,367228167)

Reply:
Filed https://github.com/dotnet/roslyn/issues/41017